### ISSUE TITLE
Optimizer: Why Gradients Rapidly Increase Near the End of Training

### ISSUE THREAD

**Issue by 67372a** [kozistr/pytorch_optimizer#388]
Paper: https://arxiv.org/abs/2506.02285
